src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Unix/CertificatePolicy.cs

```diff
@@ -86,10 +83,8 @@ internal bool MatchesCertificatePolicies(Oid policyOid)
 
                 if (policy.PolicyMapping != null)
                 {
-                    for (int iMapping = 0; iMapping < policy.PolicyMapping.Count; iMapping++)
+                    foreach (CertificatePolicyMappingAsn mapping in policy.PolicyMapping)
                     {
-                        CertificatePolicyMapping mapping = policy.PolicyMapping[iMapping];
-
```

Comment:
I guess no particular reason except that `iMapping`  was not used anywhere, so `foreach` seemed to show the intention better. I can revert this change.

Reply:
The for loop is both stack smaller (by one int and maybe one T) and more performant (by one &&, one field read, and one ==).  Certainly that level of perf doesn't really matter here (given that policy mappings are... "rare" would imply more of them than I've seen)... but since the change isn't necessary (and doesn't have a perf reason), please undo it.